core/utils/src/rootcling.cxx

```diff
@@ -4153,10 +4156,14 @@ int RootCling(int argc,
    }
 
    std::string resourceDir;
-#ifdef R__LLVMRESOURCEDIR
-   resourceDir = "@R__LLVMRESOURCEDIR@";
+#ifdef R__EXTERN_LLVMDIR
+   resourceDir = R__EXTERN_LLVMDIR;
```

Comment:
The removal of #ifdef R__LLVMRESOURCEDIR was done, as you say, because it was not used any more.

The addition of #ifdef R__EXTERN_LLVMDIR was not intended as a replacement for this, but is part of the replacement of TMetaUtils::GetLLVMResourceDir that follows. Compare to the similar replacement in core/meta/src/TCling.cxx.

Earlier in the rootcling.cxx file there is a #define R__LLVMDIR which doesn't seem to be used anywhere. Maybe this can be removed.